modules/troubleshoot/partials/debug-bundle.adoc

```diff
@@ -168,550 +170,260 @@ For a description of the files and directories, see <<Contents of the debug bund
-    "user": "",
-    "group": "redpanda"
-  },
-  "/var/lib/redpanda/data/redpanda": {
-    "size": "4.096kB",
-    "mode": "dgrwxr-xr-x",
-    "modified": "2023-02-02 15:21:10.650878326 +0000 UTC",
-    "user": "",
-    "group": "redpanda"
-  },
-  "/var/lib/redpanda/data/redpanda/controller": {
-    "size": "4.096kB",
-    "mode": "dgrwxr-xr-x",
-    "modified": "2023-02-02 15:21:10.650878326 +0000 UTC",
-    "user": "",
-    "group": "redpanda"
-  },
-  "/var/lib/redpanda/data/redpanda/controller/0_0": {
-    "size": "4.096kB",
-    "mode": "dgrwxr-xr-x",
-    "modified": "2023-02-02 15:21:12.346878368 +0000 UTC",
-    "user": "",
-    "group": "redpanda"
-  },
-  "/var/lib/redpanda/data/redpanda/controller/0_0/0-1-v1.log": {
-    "size": "4.096kB",
-    "mode": "-rw-r--r--",
-    "modified": "2023-02-02 15:21:32.450878771 +0000 UTC",
-    "user": "",
-    "group": "redpanda"
-  },
-  "/var/lib/redpanda/data/redpanda/kvstore": {
-    "size": "4.096kB",
-    "mode": "dgrwxr-xr-x",
-    "modified": "2023-02-02 15:21:10.590878324 +0000 UTC",
-    "user": "",
-    "group": "redpanda"
-  },
-  "/var/lib/redpanda/data/redpanda/kvstore/0_0": {
-    "size": "4.096kB",
-    "mode": "dgrwxr-xr-x",
-    "modified": "2023-02-02 15:21:10.602878325 +0000 UTC",
-    "user": "",
-    "group": "redpanda"
-  },
-  "/var/lib/redpanda/data/redpanda/kvstore/0_0/0-0-v1.log": {
-    "size": "8.192kB",
-    "mode": "-rw-r--r--",
-    "modified": "2023-02-02 15:21:32.458878772 +0000 UTC",
-    "user": "",
-    "group": "redpanda"
-  },
-  "/var/lib/redpanda/data/startup_log": {
-    "size": "26B",
-    "mode": "-rw-r--r--",
-    "modified": "2023-02-02 15:21:10.510878323 +0000 UTC",
-    "user": "",
-    "group": "redpanda"
-  }
-}
-```
-====
+Changes to this property take effect immediately and do not require a cluster restart.
 
-=== View cluster metadata
+Before you change this property:
 
-```bash
-cat kafka.json | jq '.[0]'
-```
+* Ensure that your chosen directory has sufficient storage capacity to handle debug bundles.
+* Verify the directory's permissions to ensure Redpanda can write to it.
 
-.Example output
-[%collapsible]
+[NOTE]
 ====
-```json
-{
-  "Name": "metadata",
-  "Response": {
-    "Cluster": "redpanda.14a3f9b6-1c74-4ffd-806a-4ab48db78120",
-    "Controller": 0,
-    "Brokers": [
-      {
-        "NodeID": 0,
-        "Port": 9093,
-        "Host": "redpanda-0.redpanda.<namespace>.svc.cluster.local.",
-        "Rack": null
-      },
-      {
-        "NodeID": 1,
-        "Port": 9093,
-        "Host": "redpanda-1.redpanda.<namespace>.svc.cluster.local.",
-        "Rack": null
-      },
-      {
-        "NodeID": 2,
-        "Port": 9093,
-        "Host": "redpanda-2.redpanda.<namespace>.svc.cluster.local.",
-        "Rack": null
-      }
-    ],
-    "Topics": {}
-  },
-  "Error": null
-}
-```
+Changing this path does not remove existing debug bundles in the original directory. If needed, you must manually remove these bundles.
```

Comment:
nit: Actually I had to double check this, but you can still remove the previous bundle even after changing the directory (internally we store the path to the bundle when it's generated and key it by the run ID).  So you don't need this note =)

Reply:
Removed